assets/js/modules/adsense/dashboard/dashboard-widget-performance.js

```diff
@@ -62,6 +62,18 @@ class AdSensePerformanceWidget extends Component {
 					maxAge: getTimeInSeconds( 'day' ),
 					context: [ 'Single', 'Dashboard' ],
 					callback: ( result ) => {
+
+						// If there are no impressions, the site is not yet displaying ads.
+						if (
+							result && result.totals &&
+							(
+								null === result.totals[2] ||
+								0    === result.totals[2]
+							)
+						) {
```

Comment:
Instead of defining a custom condition here, I think it makes sense to rely on `isDataZeroAdSense()` as well. The current check here only checks for impressions, what if there's other data that is greater than zero (see the `isDataZeroAdSense()` function you created)?

Just to clarify, of course, whatever this code here is changed to, should also apply to the below `earning-prev28days` callback logic.

Reply:
The custom condition is checking for impressions because of this component only depending on impressions to show data, will either way change to `isDataZeroAdSense()` for simplicity, have doubled check the data returned and I believe we can assume any other value returned higher than 0 implies impressions are also higher than 0, but please let me know if I'm missing anything and this assumption is wrong. 
Changed here https://github.com/google/site-kit-wp/pull/238/commits/e442b03fc85780b54de9b6e0183d09365741113e